api/interfaces/pinia.SubscriptionCallbackMutationDirect.md

```diff
@@ -6,54 +6,54 @@ sidebarDepth: 3
 
 [API Documentation](../index.md) / [pinia](../modules/pinia.md) / SubscriptionCallbackMutationDirect
 
-# Interface: SubscriptionCallbackMutationDirect
+# Interfaz: SubscriptionCallbackMutationDirect {#interface-subscriptioncallbackmutationdirect}
 
 [pinia](../modules/pinia.md).SubscriptionCallbackMutationDirect
 
-Context passed to a subscription callback when directly mutating the state of
-a store with `store.someState = newValue` or `store.$state.someState =
+Contexto pasado a un callback de suscripción cuando se muta directamente el estado de 
+un almacén con `store.someState = newValue` o `store.$state.someState =
 newValue`.
 
-## Hierarchy
+##  Jerarquía {#hierarchy}
 
 - [`_SubscriptionCallbackMutationBase`](pinia._SubscriptionCallbackMutationBase.md)
 
   ↳ **`SubscriptionCallbackMutationDirect`**
 
-## Properties
+## Propiedades {#properties}
 
-### events
+### events {#events}
 
 • **events**: `DebuggerEvent`
 
-🔴 DEV ONLY, DO NOT use for production code. Different mutation calls. Comes from
-https://vuejs.org/guide/extras/reactivity-in-depth.html#reactivity-debugging and allows to track mutations in
-devtools and plugins **during development only**.
+🔴 DEV SOLAMENTE, NO usar para código de producción. Diferentes llamadas de mutación. Viene de
```

Comment:
En vez de "DEV SOLAMENTE" veo mejor poner "SOLO PARA DESARROLLO". ¿Cómo lo ves?

Reply:
Me parece excelente, tenía duda en esa parte 😅 changing it right away